cnd/CHANGELOG.md

```diff
@@ -9,194 +9,180 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
--   `create-transaction` sub-command: Create a signed transactions for redeeming or refunding `hbit` protocols.
--   `print-secret` sub-command: Prints the secret of a swap IF the node acts in the role of Alice for this swap.
+- `create-transaction` sub-command: Create a signed transactions for redeeming or refunding `hbit` protocols.
```

Comment:
My understanding is that the release git hub action **adds* the spaces when manipulating the file.

which mean dprint removes it, release adds them, bot fight!

You also setup the release action, I guess maybe tell dlint to put 4 spaces for bullet lists?

Reply:
Yes, if you check the release workflow, I am installing dprint there and format everything after the release action updated the file to make sure there isn't a bot fight :)